pytest_playwright/pytest_playwright.py

```diff
@@ -105,9 +101,12 @@ def launch_browser(
 ) -> Callable[..., Browser]:
     def launch(**kwargs: Dict[Any, Any]) -> Browser:
         headful_option = pytestconfig.getoption("--headful")
+        slowmo_option = pytestconfig.getoption("--slowmo")
         launch_options = {**browser_type_launch_args, **kwargs}
         if headful_option:
             launch_options["headless"] = False
+        if slowmo_option:
+            launch_options["slow_mo"] = slowmo_option * 1000  # convert to second
```

Comment:
```suggestion
            launch_options["slow_mo"] = slowmo_option * 1000
```

Reply:
Are you sure that one second is ok as I find it very high and we should use milliseconds

Reply:
Ok